code/game/gamemodes/clock_cult/clock_scripture.dm

```diff
@@ -526,6 +539,7 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
 
 
 /datum/clockwork_scripture/create_object/clockwork_proselytizer //Clockwork Proselytizer: Creates a clockwork proselytizer.
+	descname = "Necessary, Converts Things"
```

Comment:
"Converts Things" might give the illusion that it's like Guvax. Perhaps "Necessary, Converts Objects"?